src/main/java/org/prebid/server/bidder/vox/VoxBidder.java

```diff
@@ -0,0 +1,83 @@
+package org.prebid.server.bidder.vox;
+
+import com.iab.openrtb.request.BidRequest;
+import com.iab.openrtb.response.Bid;
+import com.iab.openrtb.response.BidResponse;
+import io.vertx.core.http.HttpMethod;
+import org.prebid.server.bidder.Bidder;
+import org.prebid.server.bidder.model.BidderBid;
+import org.prebid.server.bidder.model.BidderCall;
+import org.prebid.server.bidder.model.BidderError;
+import org.prebid.server.bidder.model.HttpRequest;
+import org.prebid.server.bidder.model.Result;
+import org.prebid.server.exception.PreBidException;
+import org.prebid.server.json.DecodeException;
+import org.prebid.server.json.JacksonMapper;
+import org.prebid.server.proto.openrtb.ext.response.BidType;
+import org.prebid.server.util.HttpUtil;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+
+public class VoxBidder implements Bidder<BidRequest> {
+
+    private static final Map<Integer, BidType> ALLOWED_IMP_TYPES = Map.of(
+            1, BidType.banner,
+            2, BidType.video,
+            3, BidType.audio,
+            4, BidType.xNative
+    );
+    private final String endpointUrl;
+    private final JacksonMapper mapper;
+
+    public VoxBidder(String endpointUrl, JacksonMapper mapper) {
+        this.endpointUrl = HttpUtil.validateUrl(Objects.requireNonNull(endpointUrl));
+        this.mapper = Objects.requireNonNull(mapper);
+    }
+
+    @Override
+    public Result<List<HttpRequest<BidRequest>>> makeHttpRequests(BidRequest request) {
+        //todo: is it necessary to parse VoxImpExt in order to validate it?
```

Comment:
No need, if VoxImpExt checks are already done by vox.json validator,
but if you have something extra, then add it